lib/connectors/memory.js

```diff
@@ -351,26 +351,54 @@ Memory.prototype.fromDb = function(model, data) {
   if (!data) return null;
   data = deserialize(data);
   var props = this._models[model].properties;
+  function cast(prop, val, props) {
```

Comment:
Please move this function out of `fromDb`, perhaps into `Memory.prototype._castPropertyValue`?